src/task6/task6

```diff
@@ -0,0 +1,17 @@
+function reverseArray1(array) {
```

Comment:
Function names does not represent its purpose:
1. Add comments in code
or
2. Let the function name talk for itself (my preference)